### ISSUE TITLE
Contact Forms: Email error suggestion text

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/justgetsimple#55]
Recommendation from Ablr. This should be added to each form in the validation settings of each email field.

"When an incomplete email is entered the error suggestion does not provide the format of email like it does with the Website URL form field.  It would be helpful to suggest email@domain.com for the format."

**Comment by JacquelineOmollo** [comment]
done
